### PR TITLE
Fix broken python method

### PR DESCRIPTION
**Pull request by RDIL**
## PR Summary

`this` doesn’t work in python, the keyword is `self`.  

  

## PR Context  
It needs to be fixed.  

## PR Checklist

- [X] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
    - Use the present tense and imperative mood when describing your changes
- [X] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [X] [Change is not breaking](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)
- [X] [Make sure all `.h`, `.cpp`, `.cs`, `.ps1` and `.psm1` files have the correct copyright header](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [X] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
    - If the PR is work in progress, please add the prefix `WIP:` or `[ WIP ]` to the beginning of the title (the `WIP` bot will keep its status check at `Pending` while the prefix is present) and remove the prefix when the PR is ready.  
- **User-facing changes**
    - [X] Not Applicable
    - **OR**  
    - [ ] [Documentation needed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
        - [ ] Issue filed: 
- **Testing - New and feature**
    - [X] N/A or can only be tested interactively
    - **OR**
    - [X] [Make sure you've added a new test if existing tests do not effectively test the code changed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#before-submitting)
        - [X] [Add `[feature]` to your commit messages if the change is significant or affects feature tests](https://github.com/PowerShell/PowerShell/blob/master/docs/testing-guidelines/testing-guidelines.md#requesting-additional-tests-for-a-pr)

### CODE REVIEW

**Comment by iSazonov** [comment]
@RDIL Is this Python version related?

**Comment by RDIL** [comment]
@iSazonov what do you mean?  I’m just fixing broken code in this PR.

**Comment by iSazonov** [comment]
I guess that the code worked on old Python version.

**Comment by RDIL** [comment]
The code clearly defines that the script should use a python 3.x.x version, but the this keyword hasn’t been a thing since the 2.5 series.  I think it is time to update!

**Review by TylerLeonhardt — APPROVED** [review]
LGTM

File: demos/python/class1.py [hunk 1]
```diff
@@ -14,5 +16,4 @@ def method1(this):
                             1,2,3])
 
 c = returnsjson()
-print (c.method1())
-                                  
+print(c.method1())
```
Comment (TylerLeonhardt) [inline demos/python/class1.py]: Can't tell if there's a newline at the end of this file. Please make sure there is.

Comment (RDIL) [reply demos/python/class1.py]: Yeah, I made sure there was.  It would display the red emoji if there wasn't.

Comment (TylerLeonhardt) [reply demos/python/class1.py]: Cool thanks - I figured that would be the case but the `- ` on line 18 was odd

**Review by rjmholt — APPROVED** [review]
Worth noting that neither `this` nor `self` are keywords in Python; [instance methods in Python pass the instance reference explicitly through the first parameter](https://stackoverflow.com/questions/2709821/what-is-the-purpose-of-self), and you can call that parameter whatever you like (you don't even need to be consistent):

```python
class MyObject:
    def __init__(banana):
        banana.field = 10

    def get_field(truck):
         return truck.field

if __name__ == '__main__':
    print(MyObject().get_field()) # prints '10'
```

So this Python script would have run correctly using `this`.

However, `self` is the near-universal convention, and worth us aligning to that, especially in our examples. Thanks @RDIL!

**Comment by iSazonov** [comment]
@SteveL-MSFT Make sense to move this to Docs repo?

**Comment by SteveL-MSFT** [comment]
@iSazonov We may want to eventually have a doc page about interop with other languages, but we can keep this code here in this repo for now

**Pull request merged**